hack/update_bundle.sh

```
@@ -123,42 +123,35 @@ if [ -z "${RELATED_IMAGES}" ] || [ "${RELATED_IMAGES}" = "null" ]; then
-  exit 1
-fi
-PLACEHOLDERS=$(${JQ} '.' "${PLACEHOLDERS_FILE}")
-${JQ} -c '.[]' <<<"${PLACEHOLDERS}" | while IFS= read -r entry; do
-  NAME=$(${JQ} -r '.name' <<<"${entry}")
-  PLACEHOLDER=$(${JQ} -r '.placeholder' <<<"${entry}")
-  TARGET=$(${JQ} -r '.target' <<<"${entry}")
-  IMG=$(${JQ} -r --arg n "${NAME}" '.[] | select(.name==$n) | .image' <<<"${RELATED_IMAGES}")
+# Substitute deployment args and container image from related_images.json (operator_arg / operator_target).
+# shellcheck source=image_args_lib.sh
+source "${SCRIPT_DIR}/image_args_lib.sh"
+while IFS='|' read -r name placeholder target _; do
```

Shellcheck (0.11.0)

[warning] placeholder appears unused. Verify use (or export if used externally).

(SC2034)